autonomous/forward.py

```diff
@@ -8,7 +8,10 @@ class Forward(AutonomousStateMachine):
 
     drivetrain = Drivetrain
 
-    @state(first=True)
+    @state(first=True)                         '''This tells us this state is the starting state and that it is not timed.'''
     def forward(self):
-        if self.drivetrain.forward(feet=30):
+        if self.drivetrain.forward(feet=30):   
```

Comment:
Why document the function every time it is used, rather than where it is created?